### DISCUSSION TITLE
part-ii/2018-104/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#302] [Question comments]
Part II 2018 Number Theory: Paper 3, Section II, G 

https://questions.tripos.org/part-ii/2018-104/

**Comment by callumhobbis** [comment]
<details>
<summary>Sketch solution</summary>

The first bit is bookwork, and the next paragraph starts with a standard method. We then apply another standard result/method for the last part (noting $p$ is represented by some $f_i$ iff it is _properly_ represented by some $f_i$), which gets us to the point $(\tfrac p5)=(\tfrac p7)$, but we don't have the right hand side yet.

For this, note that mod 5 $f_1\equiv(x+3y)^2$ and mod 7 $f_1\equiv(x-3y)^2$, so the $n$ represented by $f_1$ are _always_ squares mod 5,7. Conversely, f_2(1,1)=7$ which is not a square mod 5, and $f_2(1,0)=3$ which is not a square mod 7.

Thus if $f_1$ represents $p$, we have $(\tfrac p5)=(\tfrac p7)=1$, and if $(\tfrac p5)=(\tfrac p7)=-1$, then it certainly can't be represented by $f_1$. This gives us the full "if and only if" statement we need. 

</details>
